src/NextApplication.java

```diff
@@ -1,2 +1,7 @@
-package src;public class NextApplication {
+package src;
+
+public class NextApplication {
+ private void dummyClass(){
```

Comment:
space between ) & {

Reply:
not nits dude 
fyi: i dont know what is nits or linters

Reply:
sathyama purila ena solra nu :/